waspc/packages/deploy/src/providers/fly/setup/setup.ts

```diff
@@ -105,9 +146,24 @@ async function setupClient(deploymentInfo: DeploymentInfo<SetupOptions>) {
 	// This creates the fly.toml file, but does not attempt to deploy.
 	await $`flyctl launch --no-deploy  ${launchArgs}`;
 
-	// goStatic listens on port 8043 by default, but the default fly.toml
-	// assumes port 8080 (or 3000, depending on flyctl version).
-	replaceLineInLocalToml(/internal_port = \d+/g, 'internal_port = 8043');
+	const internalPortOptionRegex = /internal_port = \d+/g;
+
+	if (!doesLocalTomlContainLine(internalPortOptionRegex)) {
+		await question(`⚠️  There was an issue setting up your client app.
+We tried modifying your client fly.toml to set ${boldText(
+		'internal_port = 8043',
+	)}, but couldn't find the option ${boldText(
+		'modifying',
+	)} in the fly.toml.
+
+This might mean your client app might not be accessible,
```

Comment:
```suggestion
This means your client app might not be accessible,
```

Reply:
And maybe also mention here that we advise they reach out to us.

Reply:
I mean ideally we would be notified in case any of these issues happens, automatically. To achieve something like that, we could possibly send some message to our analytics, although they are not really meant for that, what would probably make more sense is to use something like Sentry or similar. But maybe telling them to let us know is good enough for now.